src/overlays/actors/ovl_En_Hy/z_en_hy.c

```diff
@@ -6,29 +6,37 @@
 
 #include "z_en_hy.h"
 #include "objects/object_aob/object_aob.h"
+#include "objects/object_ahg/object_ahg.h"
+#include "objects/object_bob/object_bob.h"
+#include "objects/object_boj/object_boj.h"
+#include "objects/object_bba/object_bba.h"
+#include "objects/object_bji/object_bji.h"
+#include "objects/object_cne/object_cne.h"
+#include "objects/object_cob/object_cob.h"
 #include "objects/object_os_anime/object_os_anime.h"
 
 #define FLAGS 0x00000019
 
 #define THIS ((EnHy*)thisx)
 
+#define TYPE (this->actor.params & 0x7F)
+#define PATH_ID ((this->actor.params & 0x780) >> 7)
```

Comment:
same comment about this again, would prefer to not do anything yet, but should take an arg at the very least if we must have it now